src/components/Hoverable/HoverablePropTypes.js

```diff
@@ -0,0 +1,27 @@
+import {Component} from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /* Child to wrap with Hoverable. Must be only child. */
```

Comment:
`/** Child to wrap with Hoverable. Must be only child. */`

all the doc comments should look like that

Reply:
I think actually we decided we would keep the `//` for now.